src/rdkafka_conf.c

```diff
@@ -1890,6 +1899,32 @@ void rd_kafka_conf_set_open_cb (rd_kafka_conf_t *conf,
 }
 #endif
 
+rd_kafka_conf_res_t
+rd_kafka_conf_set_ssl_cert_verify_cb(rd_kafka_conf_t *conf,
+    int (*ssl_cert_verify_cb) (char *cert, size_t len, void *opaque)) {
+#ifdef __mips__
+    return RD_KAFKA_RESP_ERR__NOT_IMPLEMENTED;
```

Comment:
this is an rd_kafka_resp_err_t error code, but the method returns an rd_akfka_conf_res_t. change it to return RD_KAFKA_CONF_UNKNOWN